robot/impl/resource_manager.go

```diff
@@ -790,6 +790,10 @@ func (manager *resourceManager) completeConfigForRemotes(ctx context.Context, lr
 		manager.logger.CInfow(ctx, fmt.Sprintf("Now %s a remote", verb), "resource", resName)
 		switch resName.API {
 		case client.RemoteAPI:
+			// If a remote is marked for removal then do not process remote
+			if gNode.MarkedForRemoval() {
+				continue
+			}
```

Comment:
can we put this in front of the log? would be odd to see a log for reconfiguring log and then it being skipped. could go on L781